### ISSUE TITLE
Is there's anyway to have a case insensitive version?

### ISSUE THREAD

**Issue by haiduong87** [jasonholloway/miniguid#4]
I'm working on an storage solution, which required to create random filename.
I found this package can help me to solve the random job.
But file name is case insensitive, so this may produce some duplicate file names.

Can I just use edit this line?
https://github.com/jasonholloway/miniguid/blob/master/MiniGuid/MiniGuid.cs#L69

Duong Nguyen

**Comment by jasonholloway** [comment]
Hi Duong, you're definitely looking in the right place.

For it to work, you need 32 unique characters, which are all listed on line 69, all the characters on line 70 are just optional extras for readability purposes (including them doesn't make the encoded string any shorter/longer). If line 70 just had an empty string with no characters in it, it would all work just as well.

So, for your case, this would work I think:
```csharp
_bin2Char = "abcdefghijklmnopqrstuvwxyz012345".ToCharArray()
    .Zip("6789".PadRight(32).ToCharArray(),
        (c1, c2) => (c1, c2 != ' ' ? (char?)c2 : null))
    .ToArray();
```

Basically, you have full choice of symbols, as long as there are at least 32 of them.

I'm imagining you're wanting to fork this, but if you want to push any changes upstream I'd be happy to merge them
